tsconfig.json

```diff
@@ -3,12 +3,18 @@
     "lib": ["es2015"],
     "target": "es5",
     "module": "commonjs",
-    "declaration": true,
+    "declaration": false,
     "outDir": "./dist",
+    "allowJs": true,
     "strict": true
   },
   "include": [
     "src/*.ts",
+    "src/*.js",
```

Comment:
See above

Reply:
this tells TS to bundle `*.js` files into `dist` dir